aa-sdk/core/src/account/smartContractAccount.ts

```diff
@@ -34,14 +34,19 @@ import type { SmartAccountSigner } from "../signer/types.js";
 import { wrapSignatureWith6492 } from "../signer/utils.js";
 import type { NullAddress } from "../types.js";
 import type { IsUndefined } from "../utils/types.js";
-import { DeploymentState } from "./base.js";
```

Comment:
curious why make this change / delete this file?

Reply:
oh we're killing simple account, so I was either gonna have to document it to get things passing or just kill it. only simple account was using the old BaseSmartAccount class, so both of those had to go. The correct way to create an SCA instance is using `toSmartContractAccount`